src/test/javascript/spec/component/team/teams-import-button.component.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { DebugElement } from '@angular/core';
+import { ComponentFixture, fakeAsync, TestBed, tick, waitForAsync } from '@angular/core/testing';
+import { NgbModal, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { Team } from 'app/entities/team.model';
+import { TeamsImportButtonComponent } from 'app/exercises/shared/team/teams-import-dialog/teams-import-button.component';
+import * as chai from 'chai';
+import { SinonSpy, SinonStub, spy, stub } from 'sinon';
+import * as sinonChai from 'sinon-chai';
+import { mockExercise, mockSourceTeams, mockTeams } from '../../helpers/mocks/service/mock-team.service';
+import { config } from './teams-import-dialog.component.spec';
```

Comment:
Do not do this. Tests should not depend on the config of other tests. This blows up the dependency list. Only put into the test module what you really need